portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase

```diff
@@ -3731,6 +3731,689 @@ definition {
+		}
+
+		task ("Create two segments that the user belongs to") {
+			JSONSegmentsentry.addSegment(
+				fieldName = "First Name",
+				groupName = "Test Site Name",
+				operator = "equals",
+				segmentName = "First Segment",
+				text = "Test");
+
+			JSONSegmentsentry.addSegment(
+				fieldName = "Last Name",
+				groupName = "Test Site Name",
+				operator = "equals",
+				segmentName = "Second Segment",
+				text = "Test");
+		}
+
+		task ("Create a dynamic Collection") {
+			JSONAssetlist.addDynamicAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article",
+				title = "Dynamic Collection Test");
+		}
+
+		task ("Configure Assets to combine all segments in asset publisher") {
+			AssetListsAdmin.enableCollectionsToCombineAssetsFromSegments(collectionType = "Dynamic");
+		}
+
+		task ("Create a first variation for the collection") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "First Segment");
+
+			AssetListsAdmin.configureItemTypeInDynamicCollection(
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "First Segment");
+		}
+
+		task ("Create a second variation for the collection") {
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "Second Segment");
+
+			AssetListsAdmin.configureItemTypeInDynamicCollection(
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "Second Segment");
+		}
+
+		task ("Select the created Dynamic Collection to be used in the asset publisher portlet") {
+			Navigator.gotoSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			AssetPublisherPortlet.selectAssetList(assetListName = "Dynamic Collection Test");
+
+			IFrame.closeFrame();
+		}
+
+		task ("Assert that all web contents are displayed") {
+			AssetPublisherPortlet.viewAssetOrderPG(assetTitleList = "WC_Title_1");
+
+			AssertElementNotPresent(
+				key_assetTitle = "WC_Title_1",
+				key_index = "2",
+				locator1 = "AP#ASSET_ORDER");
+		}
+
+		task ("Set collections configuration to default values") {
+			AssetListsAdmin.resetCollectionsToCombineAssetsFromSegments();
+		}
+	}
+
+	@description = "LPS-169312: Given a user belonging to Segment 1 and Segment 2
+	And the configuration to combine assets is enabled
+	When there's a variation of a dynamic collection targeting segment 1 and segment 2, and both are prioritized above the anyone variation
+	And there is an asset present in both variations**
+	Then the user will see the asset only once in Collection Display Fragment"
+	@priority = "3"
+	test ViewCombineContentForMultisegmentDynamicCollectionInCollectionDisplayFragment {
+		property portal.acceptance = "true";
```

Comment:
This test has priority 3 so it is not considered as acceptance. Remove this property.